### PR TITLE
fix websocket connection issue in docker makeplane/plane#3195

### PR DESCRIPTION
**Pull request by abidkhan484**
Since the proxy server is running with Nginx, it is recommended to add the relevant proxy header to use Web Socket. The relevant documentation is shared below.

Reference: [Nginx WebSocket Proxying documentation](https://nginx.org/en/docs/http/websocket.html)

### CODE REVIEW

File: nginx/nginx.conf.template [hunk 1]
```diff
@@ -18,6 +18,9 @@ http {
 
         location / {
             proxy_pass http://web:3000/;
+            proxy_http_version 1.1;
```
Comment (mguptahub) [inline nginx/nginx.conf.template:21]: Move all 3 lines to nginx.conf.dev file which is used for local development

**Review by mguptahub — CHANGES_REQUESTED** [review]
Please fix and revert.

File: nginx/nginx.conf.template (hunk 1 above)
Comment (abidkhan484) [reply nginx/nginx.conf.template]: The mentioned lines are removed from the main nginx template and added to the specified local template as per instruction.

**Review by mguptahub — APPROVED** [review]
good to go

**Comment by abidkhan484** [comment]
> good to go

Thank you for your thorough review of my pull request. Your insights were incredibly helpful, and I appreciate your time and expertise.

**Review by sriramveeraghanta — APPROVED** [review]

**Pull request merged**